src/main/kotlin/no/nav/emottak/utils/ExtensionFunctions.kt

```diff
@@ -0,0 +1,9 @@
+package no.nav.emottak.utils
+
+import kotlin.uuid.Uuid
+
+fun String.parseOrGenerateUuid(): Uuid = try {
```

Comment:
Hva med denne litt mer kompakte varianten: `fun String.parseOrGenerateUuid(): Uuid = catch({ Uuid.parse(this) }) { Uuid.random() }`